en/micro-integrator/docs/references/connectors/ldap-connector/ldap-server-configuration.md

```diff
@@ -0,0 +1,60 @@
+# LDAP Connector Configuration
+
+## Setting up an LDAP Server
+WSO2 Identity Server offers an embedded LDAP as a primary user store. Download Identity Server from 
+[here](https://wso2.com/identity-and-access-management/) and start the server. See 
+[Quick Start Guide](https://is.docs.wso2.com/en/5.10.0/get-started/quick-start-guide/)
+
+### Apache Directory Studio
+
+1. Download Apache Directory Studio from [here](http://directory.apache.org/studio/) and open.
+2. Right click on the LDAP Servers tab found on the bottom left corner and click **New Connection**.
+![image](../../../assets/img/connectors/ldap_connector/ds_create_new_connection.png)
+3. Configure network parameters as follows and click next.
+![image](../../../assets/img/connectors/ldap_connector/creating_a_new_connection.png)
+4. Provide authentication parameters as follows and click finish.
+    * Bind DN or user parameter - **uid=admin,ou=system**
+    * Bind password - **admin**
+5. Right click on newly created connection and select **Open Connection**.
+![image](../../../assets/img/connectors/ldap_connector/open_connection.png)
+
+### init
+
+```xml
+<ldap.init>
+    <providerUrl>{$ctx:providerUrl}</providerUrl>
+    <securityPrincipal>{$ctx:securityPrincipal}</securityPrincipal>
+    <securityCredentials>{$ctx:securityCredentials}</securityCredentials>
+    <secureConnection>{$ctx:secureConnection}</secureConnection>
+    <disableSSLCertificateChecking>{$ctx:disableSSLCertificateChecking}</disableSSLCertificateChecking>
+</ldap.init>
+```
+
+**providerUrl** : The URL of the LDAP server.</br>
+**securityPrincipal** : The Distinguished Name(DN) of the admin of the LDAP Server.</br>
+**securityCredentials** : The password of the LDAP admin.</br>
+**secureConnection** : The boolean value for the secure connection.</br>
+**disableSSLCertificateChecking** : The boolean value to check whether certificate enable or not.</br>
+
+You can follow the following steps to import your LDAP certificate into wso2esb client’s keystore as follows:
+
+1. To encrypt the connections, we'll need to configure a certificate authority (https://www.digitalocean.com/community/tutorials/how-to-encrypt-openldap-connections-using-starttls) 
+and use it to sign the keys for the LDAP server.
+2. Use the ESB Management Console or the following command to import that certificate into the ESB client keystore. 
+```keytool -importcert -file <certificate file> -keystore <EI>/repository/resources/security/client-truststore.jks -alias "LDAP"```
```

Comment:
Separate this into multiple lines and tab it.

Reply:
Updated